### PR TITLE
Hw lesson 9

### PR DESCRIPTION
**Pull request by ranaashraf2627**

### CODE REVIEW

File: lesson9/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <h2>Course structure</h2>
+    <table>
+        <tr>
+            <th>Lesson Number</th>
+            <th>Topic</th>
+            <th>Classwork</th>
+            <th>Homework</th>
+        </tr>
+ <tr>
+<td>1</td>
```
Comment (AlexOrd) [inline lesson9/index.html]: Please use Prettier for this file.

File: lesson1/index2.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
```
Comment (AlexOrd) [inline lesson1/index2.html:1]: Good job.

**Review by AlexOrd — APPROVED** [review]

File: lesson9/index.html (hunk 1 above)
Comment (ranaashraf2627) [reply lesson9/index.html]: ok

File: lesson1/index2.html (hunk 2 above)
Comment (ranaashraf2627) [reply lesson1/index2.html:1]: thanks

**Pull request merged**